etcdctl/ctlv3/ctl.go

```diff
@@ -51,6 +53,8 @@ func init() {
 
 	rootCmd.PersistentFlags().DurationVar(&globalFlags.DialTimeout, "dial-timeout", defaultDialTimeout, "dial timeout for client connections")
 	rootCmd.PersistentFlags().DurationVar(&globalFlags.CommandTimeOut, "command-timeout", defaultCommandTimeOut, "timeout for short running command (excluding dial timeout)")
+	rootCmd.PersistentFlags().DurationVar(&globalFlags.KeepAliveTime, "keep-alive-time", defaultKeepAliveTime, "keep alive time for client connections")
```

Comment:
`s/keep-alive/keepalive/` and `s/"keep alive/"keepalive/`

Reply:
OK, I thought they are two words.